playbooks/roles/bats/defaults/main.yaml

```diff
@@ -4,6 +4,6 @@ bats_forklift_dir: "/root/forklift"
 bats_output_dir: "/root/bats_results"
 bats_update_forklift: "yes"
 bats_tests:
-  - "fb-install-katello.bats"
```

Comment:
Somewhat yes, but I feel like the latter portion is valuable: https://github.com/theforeman/forklift/blob/master/bats/fb-install-katello.bats#L58-L97